closure/library/a11y/aria/BUILD

```diff
@@ -0,0 +1,72 @@
+# DO NOT EDIT -- bazel run //closure/library:regenerate -- "$PWD"
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load("//closure:defs.bzl", "closure_js_library")
+
+closure_js_library(
+    name = "all_js",
+    exports = [
+        ":announcer",
+        ":aria",
+        ":attributes",
+        ":datatables",
+        ":roles",
+    ],
+)
+
+closure_js_library(
+    name = "announcer",
+    srcs = ["@com_google_javascript_closure_library//:closure/goog/a11y/aria/announcer.js"],
+    lenient = True,
```

Comment:
what is the strategy for what has lenient set to True vs what doesn't? the all_js don't seem to and the overall top-level build rule doesn't either.